docs/articles/documentation/guides/advanced-guides/authentication.md

```diff
@@ -76,17 +63,9 @@ const admin = Role('http://example.com/login', async t => {
 
 After you create the roles, you can switch between users at any moment except for the role initialization code.
 
-If you switch to a role for the first time in test run, the browser will be navigated from the original page to a login page where the role initialization code will be executed. Then the original page will be reloaded with new credentials (you can use the [preserveUrl](#optionspreserveurl) option to disable redirect to the original page). If you switch to a role that has already been initialized, TestCafe simply reloads the current page with the appropriate credentials.
+If you switch to a role for the first time in test run, the browser will be navigated from the original page to a login page where the role initialization code will be executed. Then the original page will be reloaded with new credentials (you can use the [preserveUrl](../../reference/test-api/role/constructor.md#optionspreserveurl) option to disable redirect to the original page). If you switch to a role that has already been initialized, TestCafe simply reloads the current page with the appropriate credentials.
 
-To switch to a role, use the `t.useRole` function.
-
-```text
-t.useRole( role )
-```
-
-Parameter | Type   | Description
---------- | ------ | ---------------------------------------------
-`role`    | Object | The role you need to use further in the test.
+To switch to a role, use the [t.useRole](../../reference/test-api/testcontroller/userole.md) function.
```

Comment:
By "To switch to a role" do you mean "To switch a role"?

Reply:
No, it's rather 'to switch the user account' or 'to activate a role'.

Reply:
Can we say:
To activate a role, use the [t.useRole](../../reference/test-api/testcontroller/userole.md) function.